src/main/menu.ts

```diff
@@ -248,7 +251,7 @@ function getFileMenu(): MenuItemConstructorOptions {
 /**
  * Creates the app's window menu.
  */
-export function setupMenu() {
+export function setupMenu(active_key: string | null = null) {
```

Comment:
```suggestion
export function setupMenu(activeKey: string | null = null) {
```

Reply:
Also since `activeKey` is unused under `index.ts`, why not use `activeKey?: string | null`